### ISSUE TITLE
Allow more than one domain for registration

### ISSUE THREAD

**Issue by t2d** [systemli/user-management#53]
Status Quo
=========

* Primary Domain
  * Has invite codes
  * Users can register with invite codes

* Secondary Domains
  * Users invite codes are hidden
  * Users have to be created by admin

Future Goals
==========
* Allow registration for multiple (but not all) domains
* Allow domain specific admins

I think we should discuss in the future how we want to classify the domains and which features each class should bring.

**Comment by 0x46616c6b** [comment]
I would prefer to add a field to the Domain entity which represents this issue. With a field we can query all domains a user can use for registration. Primary Domain sounds there could be only one domain.

**Comment by doobry-systemli** [comment]
> I would prefer to add a field to the Domain entity which represents this issue. With a field we can query all domains a user can use for registration. Primary Domain sounds there could be only one domain.

Yep, I agree. Probably something like bool `pubRegistration` to list this domain for public address registration?

**Comment by 0x46616c6b** [comment]
Implemented #1056
